dashboard/src/components/Header/Header.tsx

```diff
@@ -42,17 +42,17 @@ class Header extends React.Component<IHeaderProps, IHeaderState> {
     {
       children: "Applications",
       exact: true,
-      namespaced: true,
+      clustered: true,
       to: "apps",
     },
     {
       children: "Catalog",
-      namespaced: true,
+      clustered: false,
```

Comment:
why is the catalog not "clustered"?

Reply:
If it's something temporary it's better to leave a comment saying why.

IMO, the catalog URI should be eventually clustered:
 - "global" App repos may differ between clusters
 - namespaced repos should be eventually supported

Also, in my opinion, the multi-cluster support shouldn't be claimed as "complete" if secondary clusters have less functionality than the main one. WDYT?

Reply:
> If it's something temporary it's better to leave a comment saying why.

As above, the code no longer exists since updating to use `shared/url`.
> 
> IMO, the catalog URI should be eventually clustered:

In a following PR, you'll see that I did add the `/c/:cluster/` prefix to the catalog URI, because importantly, the URI defines the catalog available for install in that location (cluster+namespace), ie. the target.

> 
>     * "global" App repos may differ between clusters

Global app repos are global for the Kubeapps installation in the current design. If I switched to browse a different Kubeapps installed on a different cluster (which happened to also support a union of clusters as the first installation), it may have a different set of global app repos. But I don't plan for a single Kubeapps instalation to have different global app repos depending on the cluster - there is one Kubeapps installation (with a set of global app repos) which supports multiple clusters. Let's chat about this tonight?

> 
>     * namespaced repos should be eventually supported

Yep, they should, but this is tied together with deciding on the future of AppRepos with OCI in the mix, which we should do separately.

> 
> Also, in my opinion, the multi-cluster support shouldn't be claimed as "complete" if secondary clusters have less functionality than the main one. WDYT?

Do you mean in that initially it won't be possible to add a namespaced app repository to an additional cluster? If so, as above, if not - let me know :)

Reply:
> Global app repos are global for the Kubeapps installation in the current design. If I switched to browse a different Kubeapps installed on a different cluster (which happened to also support a union of clusters as the first installation), it may have a different set of global app repos. But I don't plan for a single Kubeapps instalation to have different global app repos depending on the cluster - there is one Kubeapps installation (with a set of global app repos) which supports multiple clusters. Let's chat about this tonight?

I think that's a valid use case that we should take into account (even if we don't implement it for the moment). As administrator, I may want to enable certain repositories for everyone with access to one cluster and other repos for a second cluster (in a dev vs prod environment for example).

Reply:
Definitely - I just mean that I would do that as part of the *one* Kubeapps installation - enabling the configuration of different repos on different clusters as part of the one instance of Kubeapps' configuration.